vtex-apps/powerreviews
# [POWEREVIEW-2] create order in power reviews

**What problem is this solving?**

Sends an email (after a day, or whatever the given interval is) when orders are invoiced

**How should this be manually tested?**

Go to settings here: https://weiworkspace--sandboxusdev.myvtex.com/admin/apps/vtex.powerreviews@2.3.3/setup/
Check **Send Email After Invoicing**
Make sure broadcaster is pointed to my workspace
Place an order. Invoice it. Wait a day
